### PR TITLE
Remove an unused type BlobDoesNotExist

### PR DESCRIPTION
**Pull request by ihcomega56**
## Description

Related to pyrsia/pyrsia#1221

Deleted one of error types `BlobDoesNotExist` which is never used.
Let me know if it should not be removed.

## Screenshots

Also some relevant documents are updated in this PR.

image

image

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/community/get_involved/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).

### Code Contributions

- [x] I've built the code `cargo build --all-targets` successfully.
- [x] I've run the unit tests `cargo test --workspace` and all tests pass.
- [x] I've run the [integrations tests](https://github.com/pyrsia/pyrsia-integration-tests#how-to-set-up-and-run-the-tests) and all tests passes.

### CODE REVIEW

**Review by dasmanas — APPROVED** [review]
lgtm

File: pyrsia_node/readme.md [hunk 1]
```diff
@@ -1,32 +0,0 @@
-# Pyrsia Node
```
Comment (ihcomega56) [inline pyrsia_node/readme.md:1]: Moved the documents to pages in Pyrsia.io.

File: docs/tutorials/docker.md [hunk 2]
```diff
@@ -249,3 +249,27 @@ pyrsia build docker --image alpine:3.16.3
 This will send the build request to one of the authorized build nodes, which will
 start a build. When consensus about that build is reached, it will be available
 in the transparency logs and can be retrieved by all nodes in the network.
+
+# Use API
+
+Pyrsia supports Docker Registry API. The entire API documents will be coming soon.
```
Comment (ihcomega56) [inline docs/tutorials/docker.md:255]: > The entire API documents will be coming soon.

Is it correct?

Comment (erwin1) [reply docs/tutorials/docker.md:255]: the intention is that we document this API yes. although the registry API is of course already documented: https://docs.docker.com/registry/spec/api/

However, Pyrsia only supports a subset of the registry API

```suggestion
Pyrsia supports a subset of the [Docker Registry API](https://docs.docker.com/registry/spec/api/). A more detailed API document specific to Pyrsia, will be coming soon.
```

Comment (ihcomega56) [reply docs/tutorials/docker.md:255]: Thank you. That is what I wanted to know 🙏

**Review by erwin1 — APPROVED** [review]

**Review by fishseabowl — APPROVED** [review]
lgtm. Additionally, code coverage dropped by 1.76%. It looks odd.

**Pull request merged**